src/postgres/mod.rs

```diff
@@ -125,8 +125,8 @@ impl Image for Postgres {
 
     fn ready_conditions(&self) -> Vec<WaitFor> {
         vec![
-            WaitFor::message_on_stderr("database system is ready to accept connections"),
-            WaitFor::message_on_stdout("database system is ready to accept connections"),
+            WaitFor::message_on_either_std("listening on IPv4 address \"0.0.0.0\", port 5432"),
```

Comment:
port can be modified by users in general

Reply:
Updated with a suggestion to truncate the log line, and remove the IP or port